Formula/libtiff.rb

```diff
@@ -21,19 +21,57 @@ class Libtiff < Formula
 
   # All of these have been reported upstream & should
   # be fixed in the next release, but please check.
+  # Patches are taken from latest Fedora package, which is currently
+  # libtiff-4.0.9-13.fc30.src.rpm
```

Comment:
```suggestion
  # libtiff-4.0.9-13.fc30.src.rpm and whose changelog can be found at:
  # https://apps.fedoraproject.org/packages/libtiff/changelog/
```

It took me a long time to find the changelog for Fedora’s `fc30` revision; they didn’t turn up at several Google searches. The revision is also [not even included in Fedora’s Git repo](https://src.fedoraproject.org/cgit/rpms/libtiff.git) yet. So I feel the full URL of the changelog should be included via comment.